sdk/identity/identity/test/manual/README.md

```diff
@@ -18,9 +18,22 @@ Webpack will compile the code and then host it in a local server at
 `http://localhost:8080`.  See the section below about CORS before attempting to
 navigate to this URL.
 
-You will need an AAD App Registration that has its redirect uri set to
-`http://localhost:8080`.  Its tenant ID and client (application) ID will need to
-be entered into the UI to initiate the authentication process.
+You will need to configure an AAD App Registration as follows:
+
+- Create a new AAD App Registration.
```

Comment:
I've made a new issue to change this to make use of the CLI: https://github.com/Azure/azure-sdk-for-js/issues/12961